### DISCUSSION TITLE
The placement of ORFS failed with fixed pins location

### DISCUSSION THREAD

**Discussion by 706697916** [The-OpenROAD-Project/OpenROAD-flow-scripts#2129] [General] [answered]
I have a tcl text to fix the location of Pins in the layout of ORFS. If all of the Pins is fixed, the placement of ORFS failed, and if I remove some of the fixed pins, the ORFS can work well. Below is the report of the ORFS failure. How can I solve this problem with fixed location of all the Pins.
![image](https://github.com/The-OpenROAD-Project/OpenROAD-flow-scripts/assets/70256434/35c82092-3db4-4e96-a3a4-108edb26edde)

**Comment by maliberty** [comment]
Can you provide a test case?

**Reply by 706697916** [reply, marked as answer]
I figured out the problem. The area between core and die is not enough for the routing of all fixed pins.
